nucliadb/nucliadb/search/search/query.py

```diff
@@ -51,9 +51,47 @@
 from nucliadb_utils import const
 from nucliadb_utils.utilities import has_feature
 
+from .exceptions import InvalidQueryError
+
 ENTITY_FILTER_PREFIX = "/e/"
 LABEL_FILTER_PREFIX = "/l/"
 
+LABEL_QUERY_ALIASES = {
+    # aliases to make querying labels easier
+    "tags": ["t"],
+    "labels": ["l"],
+    "entities": ["e"],
+    "icon": ["n", "i"],
+    "status": ["n", "s"],
+    "primary-language": ["s", "p"],
+    "secondary-languages": ["s", "s"],
+    "field": ["f"],
+    "field-values": ["fg"],
+    "metadata": ["m"],
```

Comment:
thought: should we use `origin-metadata` instead? to be more specific.

Reply:
I was also thinking of something that mapped directly to the API like:

- "origin/tags"
- "classification/labels"
- "metadata/status"
- "metadata/language"
- "metadata/languages"
- "origin/metadata"